src/Core/Mesh/AccessDataInCells/Code.cxx

```diff
@@ -0,0 +1,82 @@
+/*=========================================================================
+ *
+ *  Copyright Insight Software Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkMesh.h"
+#include "itkLineCell.h"
+
+int main(int, char *[])
+{
+  typedef float                     PixelType;
```

Comment:
This could should be updated based on the source:
https://github.com/InsightSoftwareConsortium/ITK/blob/047fa6934ac32711d1d7b84fa6eb81c72db01960/Examples/DataRepresentation/Mesh/Mesh3.cxx
In particular, `using` instead of `typedef`, but many other style changes too. A lot has changed over the last 4 years, since this PR was started 😄